tests/test_pipelines.py

```diff
@@ -38,34 +40,35 @@
 ############################################
 
 
-class PipelinesBase:
-    def setUp(self):
+class PipelinesBase(unittest.TestCase, ABC):
+    def setUp(self) -> None:
         self.tmp_path = self.setup_output_dir()
         self.script = self.setup_script()
         self.cmd_parameters = self.setup_cmd_parameters()
 
-    def tearDown(self):
-        shutil.rmtree(self.tmp_path)
+    def tearDown(self) -> None:
```

Comment:
here you use some checks before the rmtree, in other test_* files not. should we adapt it everywhere?

Reply:
not really needed in th eother tests as the folders are always created in the init. but in pipeliens, this can go wrong when to dir_output is not ptoperly configured in teh config